roles/customize_home/tasks/main.yml

```diff
@@ -20,6 +20,9 @@
   when: customize_home_git_repo is defined
 
 - include_tasks: fix_env.yml
+  args:
+    apply:
+      become: false
```

Comment:
Hmmm, when I tried it without `become: false` it tried to use `/root/` but with `become: false` it fails due to permission denied.

I will have to try `/home/{{ ansible_user }}/foreman/.env` instead